lib/matplotlib/patches.py

```diff
@@ -1066,6 +1066,123 @@ def set_data(self, values=None, edges=None, baseline=None):
         self.stale = True
 
 
+class Lines(Patch):
+    """A semiplane bounded by a polyline on one side."""
+    def __str__(self):
+        if len(self._path.vertices):
+            s = "Line%d((%g, %g) ...)"
+            return s % (len(self._path.vertices), *self._path.vertices[0])
+        else:
+            return "Line0()"
+
+    @_docstring.dedent_interpd
+    def __init__(self, xy, direction, **kwargs):
+        """
+        *xy* is a numpy array with shape Nx2.
+
+        If *direction* is *True*, the patch is lower bounded by the polyline.
```

Comment:
I wonder if there is a better name for `direction`? `lowerbounded`? `lower`? `lower_bounded`?